### PR TITLE
Person card (full ver) Part 1/2: UI 

### PR DESCRIPTION
**Pull request by lin8231**
* Create PersonCard component
* Implemented UI
* I've split this PR up into 2 so that it doesn't get too big, logic/rest of the person card to come in the next PR
  * Currently everything is hardcoded
    * Bottom right icons will be social medias
* Added a snapshot test
  * Enzyme has no official React-17 adapter, using React's version 'react-test-render' instead.
* Added hover animation 0.3s
  * yellow when hovered (screenshot below shows hovered state)
* MoreHoriz icon is animated thx MUI 
* Additional .css file (ontop of .module.css file) is necessary in this particular case for accessing and manipulating a MUI root element 

image

Part 1/2 of #60

### CODE REVIEW

File: forgettable-frontend/src/components/PersonCard/PersonCard.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.MuiAvatarGroup-avatar {
```
Comment (noahjade) [inline forgettable-frontend/src/components/PersonCard/PersonCard.css:1]: oh this is sooooo much cleaner than the way i got around this :0 i love it!

**Review by noahjade — APPROVED** [review]
Such amazing work!! ✨ ✨ Love the snapshot testing too!

**Comment by hajineats** [comment]
Removed link to #60 since it shouldn't be closed yet

**Review by hajineats — APPROVED** [review]
Looks good!

**Pull request merged**